CONTRIBUTING.md

```diff
@@ -87,69 +87,158 @@ You may need to install the dev_requirements for this
 pip install -r path/to/source/dev_requirements
 ```
 
-_Hub:_  
-`pytest azext_iot/tests/iothub/test_iot_ext_unit.py`
+Example unit tests runs:
 
-_DPS:_  
-`pytest azext_iot/tests/dps/test_iot_dps_unit.py`
+_Hub:_
+`pytest azext_iot/tests/iothub/core/test_iot_ext_unit.py`
+
+_DPS:_
+`pytest azext_iot/tests/dps/core/test_dps_discovery_unit.py`
+
+Unit tests end in "_unit" so execute the following command to run all unit tests,
+`pytest -k "_unit"`
```

Comment:
Should the unit test examples not have the file extension based on the concern raised on the integration test examples? I think they should be consistent.